draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -5299,7 +5299,7 @@ Proxy-Authorization: basic aGVsbG86d29ybGQ=
    A <x:dfn>100-continue</x:dfn> expectation informs recipients that the
    client is about to send a (presumably large) message body in this request
    and wishes to receive a <x:ref>100 (Continue)</x:ref> interim response if
-   the request-line and header fields are not sufficient to cause an immediate
+   the method, target URI and header fields are not sufficient to cause an immediate
```

Comment:
Directly below you use an Oxford comma. You probably want to be consistent (and add the comma 😊). Same below a few times.

Reply:
Done.